pyecoregen/templates/module.py.tpl

```diff
@@ -3,9 +3,9 @@
 from functools import partial
 import pyecore.ecore as Ecore
 from pyecore.ecore import *
-{% for c in imported_classifiers -%}
-    from {{ c.ePackage | pyfqn }} import {{ c.name }}
-{% endfor %}
+{% for package, classifs in imported_classifiers.items() -%}
+    from {{ package | pyfqn }} import {{ classifs|map(attribute='name')|join(', ') }}
```

Comment:
Please separate the filter expressions by spaces around the bar as done in the other template code.

Reply:
Done!